tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,43 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _wait_element = (By.CLASS_NAME, 'collections')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(EC.presence_of_all_elements_located(
+            (self._wait_element)))
+        return self
+
+    @property
+    def featured(self):
```

Comment:
Oh whoops, I somehow thought this was referencing the themes page.

Yeah they aren't regions at all.